tracpro/contacts/migrations/0017_auto_20170915_2217.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contacts', '0016_auto_20170913_1328'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='contact',
+            name='group',
+        ),
+        migrations.AlterField(
+            model_name='contact',
+            name='groups',
+            field=models.ManyToManyField(help_text='All cohorts to which this contact belongs.', related_name='contacts', verbose_name='Cohorts', to='groups.Group'),
```

Comment:
The convention is not to capitalize verbose names. Django will capitalize them when needed.